README.md

```diff
@@ -141,9 +141,25 @@ Now audit your cluster. If someone else made this change, would you be able to p
 <img src="./example_images/changetracking.png">
 </details>
 
-### Drift and Namespace Comparison
+### Deployment Image Change Tracking
+
+Create an nginx deployment. Then change the image tag to simulate an unexpected image tag change.
 
+```shell
+kubectl apply -f https://raw.githubusercontent.com/robusta-dev/kubernetes-demos/deployment_image_change/before_image_change.yaml
+kubectl apply -f https://raw.githubusercontent.com/robusta-dev/kubernetes-demos/deployment_image_change/after_image_change.yaml
 ```
+Did you immediately get notified about a change in the image tag?
```

Comment:
This will only work if you configure the playbook first, right?

Can we link to the tutorial to configure the relevant playbook.

Reply:
Yes that's right. But I need this merged for the tutorial to work. Will link it from here after the tutorial is merged.

Reply:
Just say *something* here so that it's correct in its current form. ("You will need to configure a playbook... instructions coming soon!")

Reply:
After changing, please go ahead and merge.